### DISCUSSION TITLE
How to resume project without upgrading

### DISCUSSION THREAD

**Discussion by rabin-thami** [supabase/supabase#28486] [Questions]
I am the paid user of last month now and my subscription end's today and my project is in pause how can i resume that  without upgrade

**Comment by GaryAustin1** [comment]
You could contact support and see if there is a way.  Pause/restore always upgrades as your instance does not exist anymore and a new one is setup is my understanding.   Not sure how a paid instance gets paused though.

**Reply by rabin-thami** [reply]
OK
